### PR TITLE
feat(tests): add 2 test scopes and examples

### PR DESCRIPTION
**Pull request by ludmilanesvitiy**
# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [ ] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [ ] built and tested the changes locally.
 - [ ] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -163,6 +169,16 @@ jobs:
...
 
+#     test cypress full for herokuapp
+    - stage: cypress-tests:full:herokuapp
+      env: URL=https://ngx-universal.herokuapp.com/#
+      script: $(npm bin)/wait-on http-get://$URL && npm run cy:run:smoke && npm run cy:run:full
+
+#     test cypress full for gh-pages
+    - stage: cypress-tests:full:gh-pages
+      env: URL=https://valor-software.com/ngx-bootstrap/#/
+      script: $(npm bin)/wait-on http-get://$URL && npm run cy:run:smoke && npm run cy:run:full
```
Comment (ludmilanesvitiy) [inline .travis.yml]:
```suggestion
      npm run cy:run:smoke && npm run cy:run:full
```

File: .travis.yml [hunk 2]
```diff
@@ -163,6 +169,16 @@ jobs:
         on:
           tags: true
 
+#     test cypress full for herokuapp
+    - stage: cypress-tests:full:herokuapp
+      env: URL=https://ngx-universal.herokuapp.com/#
+      script: $(npm bin)/wait-on http-get://$URL && npm run cy:run:smoke && npm run cy:run:full
```
Comment (ludmilanesvitiy) [inline .travis.yml]:
```suggestion
      npm run cy:run:smoke && npm run cy:run:full
```

File: cypress/full/carousel_page_spec.ts [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+    });
+
+    it('When user change the interval in input to any negative value - then carousel stopped', () => {
+      carousel.scrollToMenu('Interval');
+      const newInterval = '-100';
+      carousel.clearInputAndSendKeys(intervalSlides, newInterval);
+      carousel.isInputHaveAttrs(intervalSlides,
+        [{ attr: 'type', value: 'number' }, { attr: 'ng-reflect-model', value: newInterval }]);
+      carousel.isCarouselItemActive(intervalSlides, 0);
+      cy.wait(3000);
```
Comment (ludmilanesvitiy) [inline cypress/full/carousel_page_spec.ts]: cy tick?

File: cypress/support/carousel.po.ts [hunk 4]
```diff
@@ -12,6 +12,69 @@ export class CarouselPo extends BaseComponent {
...
+      .should('to.have.descendants', this.indicatorClass)
+      .and('to.have.descendants', this.itemClass)
+      .and('to.have.descendants', this.leftControl)
+      .and('to.have.descendants', this.rightControl);
+  }
+
+  isEachSlideHave(baseSelector: string, slideParams: string[]) {
+    cy.get(`${baseSelector} ${this.carouselClass} slide `)
+      .each((slide, index) => {
+        for (let i; i < slideParams.length; i++) {
```
Comment (ludmilanesvitiy) [inline cypress/support/carousel.po.ts]:
```suggestion
        let i = 0;
        for (; i < slideParams.length; i++) {
```

File: .travis.yml (hunk 2 above)
Comment (ludmilanesvitiy) [inline .travis.yml]:
```suggestion
      npm run cy:run:smoke && npm run cy:run:full
```

File: cypress/support/carousel.po.ts (hunk 4 above)
Comment (ludmilanesvitiy) [inline cypress/support/carousel.po.ts]:
```suggestion
        let i = 0;
        for (; i < slideParams.length; i++) {
```

**Review by valorkin — APPROVED** [review]

**Pull request merged**